README.md

```diff
@@ -208,7 +208,6 @@ brew install libsodium zmq
 ```
 
 ## Integration Tests
-(Optional): If you want to recreate fake certs for your test: `make generate-certs`
```

Comment:
`generate-certs` is called by default when running integration tests https://github.com/teslamotors/fleet-telemetry/blob/main/Makefile#L51